src/main/java/io/quarkus/bot/RemoveInvalidLabelOnReopen.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.event.Issue;
+import io.quarkiverse.githubapp.event.PullRequest;
+import io.quarkus.bot.config.QuarkusBotConfig;
+import io.quarkus.bot.util.Labels;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHPullRequest;
+
+import javax.inject.Inject;
+import java.io.IOException;
+
+class RemoveInvalidLabelOnReopenAction {
+    private static final Logger LOG = Logger.getLogger(RemoveInvalidLabelOnReopenAction.class);
+
+    @Inject
+    QuarkusBotConfig quarkusBotConfig;
+
+    public void onIssueReopen(@Issue.Reopened GHEventPayload.Issue issuePayload) throws IOException {
+
+        GHIssue issue = issuePayload.getIssue();
+        if (!quarkusBotConfig.isDryRun()) {
+            issue.removeLabel(Labels.TRIAGE_INVALID);
```

Comment:
Could you check if the label is present before trying to remove it? That would avoid an API call.

Please do it for the whole block so that we don't log anything in dry run if the label isn't around.